Revive Deeper Network Open Grant application

fyang1024 opened the pull request:
### Project Abstract

This application is a follow-up to the previous deeper network open grant, https://github.com/w3f/Grants-Program/pull/141
The previous application was terminated recently due to lack of update and response from the team. The reason was the Chao Ma who submitted the original application left the team in May and the application has been overlooked. And the team has been busy preparing deeper network mainnet launch and didn't respond the termination warning in time.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying? 
- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/w3f/Grants-Program?pullRequest=581) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/w3f/Grants-Program?pullRequest=581) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/w3f/Grants-Program?pullRequest=581) it.</sub>

alxs commented:
Thanks @fyang1024. Could you please sign the CLA, and update the costs to be denominated in USD? We can still pay out in BTC or USDT/DAI if you prefer, but the amount now needs to be in USD.

fyang1024 commented:
> Thanks @fyang1024. Could you please sign the CLA, and update the costs to be denominated in USD? We can still pay out in BTC or USDT/DAI if you prefer, but the amount now needs to be indicated in USD.
> 
> Besides, please explicitly mention that the starting date will be that of the amendment, and as @Noc2 mentioned briefly share the reasons behind the current delay.

Hi @alxs, I have signed the CLA, updated the costs to be in USD, and also added a note about the starting date and reasons of delay. Thank you.

Noc2 reviewed (approved):
Looks good to me. Thanks
